authors/apps/profiles/serializers.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework import serializers
+from .models import Profile
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+    username = serializers.SerializerMethodField()
+
+    first_name = serializers.CharField(allow_blank=True, required=False,
+                                       min_length=2, max_length=50)
+    last_name = serializers.CharField(allow_blank=True, required=False,
+                                      min_length=2, max_length=50)
+    bio = serializers.CharField(allow_blank=True, required=False)
+
+    image = serializers.URLField(allow_blank=True, required=False)
```

Comment:
@Rhytah , Why are you using `serializers.URLField()` on the image and not `serializers.ImageField()`.